spec/forms/register_nonprofit_form/stripe_account_form_spec.rb

```diff
@@ -1,61 +1,54 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 require "rails_helper"
 
-RSpec.describe RegisterNonprofitForm::StripeAccountForm, :type => :model do
-
-  around(:each) do |ex|
+RSpec.describe RegisterNonprofitForm::StripeAccountForm, type: :model do
+  around do |ex|
     StripeMockHelper.mock do
       ex.run
     end
   end
 
-  context 'validation' do
-    it 'accepts a nonprofit' do
+  describe "validation" do
+    it "accepts a nonprofit" do
       form = described_class.new(nonprofit: build(:nonprofit))
       expect(form).to be_valid
     end
 
-    it 'rejects a string' do
+    it "rejects nil" do
       form = described_class.new(nonprofit: nil)
-      expect(form).to_not be_valid
+      expect(form).not_to be_valid
     end
 
-    it 'rejects a string' do
+    it "rejects a string" do
       form = described_class.new(nonprofit: "a string")
-      expect(form).to_not be_valid
+      expect(form).not_to be_valid
     end
   end
 
-
-  describe "#save" do 
+  describe "#save" do
     it "sets error when nonprofit is not persisted" do
       form = described_class.new(nonprofit: build(:nonprofit))
       expect(form.save).to be false
 
-      expect(form.errors).to_not be_empty
+      expect(form.errors).not_to be_empty
     end
 
-    it 'sets error when StripeAccountUtils.create returns a nil' do
-      
-
+    it "sets error when StripeAccountUtils.create returns a nil" do
       form = described_class.new(nonprofit: create(:nonprofit))
-      expect(StripeAccountUtils).to receive(:create).and_return nil
+      allow(StripeAccountUtils).to receive(:create).and_return nil
```

Comment:
`allow` and `expect` do different things there. This change removes a desired expectation from the spec.

Reply:
I agree they mean different things and always wondered why rubocop is so adamant about using allow over expect.  I can change it back but in the future it will fail standardrb and we'll have to update it then.